cfgov/core/tests/test_management_inactive_users.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from StringIO import StringIO
+
+from datetime import timedelta
+
+import mock
```

Comment:
I always waffle on this because for us mock is a third-party import until we're at Python 3 (then it's in `unittest.mock` anyway).